tests/TestCase/View/Helper/DocBlockHelperTest.php

```diff
@@ -0,0 +1,183 @@
+    {
+        $className = 'Comments';
+        $classType = 'Model';
+        $lines = [
+            'Line 1',
+            '@foo $bar baz',
+            '@see there',
+        ];
+        $classDescription = $this->DocBlockHelper->classDescription($className, $classType, $lines);
+        $expected = "/**\n * Comments Model\n *\n * Line 1\n * @foo \$bar baz\n *\n * @see there\n */";
+        $this::assertSame($expected, $classDescription);
+    }
+
+    /**
+     * Tests the associatedEntityTypeToHintType method
+     *
+     * @return void
+     * @covers ::associatedEntityTypeToHintType
+     */
+    public function testAssociatedEntityTypeToHintType()
+    {
+        // Test with MANY_TO_MANY
+        $type = 'Foo';
+        $association = new BelongsToMany('Foo');
+        $assocEntityType = $this->DocBlockHelper->associatedEntityTypeToHintType($type, $association);
+        $expected = 'Foo[]';
+        $this->assertSame($expected, $assocEntityType);
+
+        // Test with ONE_TO_MANY
+        $type = 'Bar';
+        $association = new HasMany('Bar');
+        $assocEntityType = $this->DocBlockHelper->associatedEntityTypeToHintType($type, $association);
+        $expected = 'Bar[]';
+        $this->assertSame($expected, $assocEntityType);
+
+        // Test with ONE_TO_ONE
+        $type = 'Ping';
+        $association = new HasOne('Ping');
+        $assocEntityType = $this->DocBlockHelper->associatedEntityTypeToHintType($type, $association);
+        $expected = 'Ping';
+        $this->assertSame($expected, $assocEntityType);
+
+        // Test with MANY_TO_ONE
+        $type = 'Pong';
+        $association = new BelongsTo('Pong');
+        $assocEntityType = $this->DocBlockHelper->associatedEntityTypeToHintType($type, $association);
+        $expected = 'Pong';
+        $this->assertSame($expected, $assocEntityType);
+    }
+
+    /**
+     * Tests the buildEntityPropertyHintTypeMap method
+     *
+     * @return void
+     * @covers ::buildEntityPropertyHintTypeMap
+     */
+    public function testBuildEntityPropertyHintTypeMap()
+    {
+        $this->markTestIncomplete('Not implemented yet');
+    }
+
+    /**
+     * Tests the buildEntityAssociationHintTypeMap method
+     *
+     * @return void
+     * @covers ::buildEntityAssociationHintTypeMap
+     * @covers ::_insertAfter
+     */
+    public function testBuildEntityAssociationHintTypeMap()
+    {
+        $this->markTestIncomplete('Not implemented yet');
+    }
+
+    /**
+     * Tests the columnTypeToHintType method
+     *
+     * @return void
+     * @cvoers ::columnTypeToHintType
+     */
+    public function testColumnTypeToHintType()
+    {
+        $this->markTestIncomplete('Not implemented yet');
+    }
+
+    /**
+     * Tests the propertyHints method
+     *
+     * @return void
+     * @covers ::propertyHints
+     */
+    public function testPropertyHints()
+    {
+        $this->markTestIncomplete('Not implemented yet');
```

Comment:
Can we implement these or delete the tests?

Reply:
Sure we can, either. I just had no time yet. I just wanted to increase the visibility for the lack of coverage in the mean time.